### PR TITLE
message added when submitter does not exists

### LINKED ISSUE #2153: Submitter: undefined undefined
**Describe the bug**
When there is not any submitter, an "undefined undefined" message appears.
image

**To Reproduce**
I am not pretty sure how to reproduce this error, but when i login as admin and i go to: 
https://demo7.dspace.org/mydspace?configuration=workflow

**Expected behavior**
The submitter should not appear if not exists or a message with "N/A". Maybe ussing the translation pipe

### PR DESCRIPTION
**Pull request by CrisGuzmanS**
Hello @tdonohue ✋ i finished this issue, i hope it is worth for DSpace.

## References
Fixes #2153  

## Description
When there is no submitter an specific message will be displayed instead of a technical words "undefined undefined"

## Instructions for Reviewers
I just handled the asynchronous function for 

List of changes in this PR:
* "submitter.empty": "N/A" in en.json5

**Include guidance for how to test or review your PR.** This may include: steps to reproduce a bug, screenshots or description of a new feature, or reasons behind specific changes. 

## Checklist

- [x] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [ ] My PR passes [ESLint](https://eslint.org/) validation using `yarn lint`
- [x] My PR doesn't introduce circular dependencies (verified via `yarn check-circ-deps`)
- [x] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [ ] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [x] If my PR includes new libraries/dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.
- [x] If my PR includes new features or configurations, I've provided basic technical documentation in the PR itself.
- [x] If my PR fixes an issue ticket, I've [linked them together](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

### CODE REVIEW

**Comment by CrisGuzmanS** [comment]
Tim let me fix the errors

File: src/app/shared/object-collection/shared/mydspace-item-submitter/item-submitter.component.spec.ts [hunk 1]
```diff
@@ -60,10 +60,17 @@ describe('ItemSubmitterComponent', () => {
...
 
-  it('should show a badge with submitter name', () => {
+  it('should show a badge with submitter name', fakeAsync(() => {
+
     const badge = fixture.debugElement.query(By.css('.badge'));
 
-    expect(badge).not.toBeNull();
-    expect(badge.nativeElement.innerHTML).toBe(EPersonMock.name);
-  });
+    fixture.whenStable().then(() => {
```
Comment (atarix83) [inline src/app/shared/object-collection/shared/mydspace-item-submitter/item-submitter.component.spec.ts:67]: why have you changed in this way ? the previous way works as well

moreover can you add a test to check the `N/A` value? thanks

**Review by atarix83 — CHANGES_REQUESTED** [review]
thanks @CrisGuzmanS 

I've reviewed it, can you check my feedback? thanks

**Comment by CrisGuzmanS** [comment]
Ok @tdonohue there is no problem, if https://github.com/DSpace/dspace-angular/pull/2129 solves the problem this pull request can be closed 👍

**Comment by tdonohue** [comment]
Closing, as I've verified the original bug is fixed in #2129, so this PR is unnecessary.  A screenshot is available in my review: https://github.com/DSpace/dspace-angular/pull/2129#pullrequestreview-1400675873

**Pull request closed without merging**